source/manual/merge-pr.html.md

```diff
@@ -55,16 +55,14 @@ You should also thank the contributor with an amount of emoji proportional to th
 Finally, clean up by deleting the branch you created.
 
 ```bash
-git pull
+brew install hub
 
-git fetch https://github.com/thomasjhughes/publisher patch-1:thomasjhughes-patch-1
+hub pr checkout <PR-NUMBER>
```

Comment:
I assume this has the downside of changing the working tree?

Reply:
Good point. Yes.